doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-supported-permissions-ref.adoc

```diff
@@ -0,0 +1,97 @@
+[id='security-management-rest-api-supported-permissions-ref_{context}']
+= Supported permissions in {CENTRAL}
+
+The following are available permissions in {PRODUCT}. Administrators use these permissions to allow specific actions to a group, role, or user in {CENTRAL}.
+
+Priority::
+Priority is an integer that defines the precedence of users who are assigned multiple roles or groups. The default value of priority for a new group is `-100`. In {CENTRAL}, you can set an integer value as a priority, which is resolved using the following rules:
++
+--
+.Priority value table
+[cols="50%,50%", frame="all", options="header"]
+|===
+|Integer value
+|Priority
+
+|Between -5 and 0
+|LOW
+
+|Less than -5
+|VERY LOW
+
+|Equals to 0
+|NORMAL
+
+|Between 0 and 5
+|HIGH
+
+|Greater than 5
+|VERY HIGH
+|===
+--
+
+Home Page::
+`homePage` indicates the default landing page for users.
+
+Workbench::
+Workbench consists of the following defined permissions:
++
+--
+[source,json]
+----
+{
+	"editDataObject": true,
+	"plannerAvailable": true,
+	"editGlobalPreferences": true,
+	"editProfilePreferences": true,
+	"accessDataTransfer": true,
+	"jarDownload": true,
+	"editGuidedDecisionTableColumns": true
+}
+----
+--
+
+Pages, Editor, Spaces, and Projects::
+The following are possible values for the permissions based on the resource type:
++
+--
+* `PAGES: read,create,update,delete`
+* `EDITOR: read`
+* `SPACES: read,update,delete,create`
+* `PROJECT: read,create,update,delete,build`
+
+You can use following code to add exceptions to *Pages*, *Editor*, *Spaces*, and *Projects* permissions:
+
+[source,json]
+----
+[{
+	"name": "HomePerspective",
+	"permissions": {
+		"create": true,
+		"read": true,
+		"delete": false
+	}
+}]
+----
+The `name` attribute is an identifier of a resource that you add as an exception. Use the following REST API endpoints to get the list of possible identifiers. The REST API base URL is `http://SERVER:PORT/business-central/rest/`.
+
+* `[GET] /perspectives`: Returns all pages in {CENTRAL}.
+* `[GET] /editors`: Returns all editors in {CENTRAL}.
+* `[GET] /spaces`: Returns all spaces in {CENTRAL}.
+* `[GET] /spaces/{spaceName}/projects`: Returns projects in a specified space.
+
+.Example object (JSON)
+[source,json]
```

Comment:
Do you mean "Example server response (JSON)" here? You say Example object, which is inconsistent. If so, I'd say "Example server response for pages (JSON)".

Also, the previous list of endpoints says `[GET] /perspectives` for pages, but the example says `"pages": { ...`. And the list before that says `PAGES: create,read,update, ...`. So is that `/perspectives` endpoint for sure accurate? Or should it be `[GET] /pages`? Maybe check with SME?

Reply:
@sterobin Thanks,  will change it as suggested. 

Regarding the `/perspective`, Perspectives is more of a business central jargon for pages, as confirmed with @RishiRajAnand, therefore will be changing the description as "Returns perspective names of all pages in {CENTRAL}". 
WDYT?